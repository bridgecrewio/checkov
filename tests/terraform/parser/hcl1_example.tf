# Example of HCL1 syntax that won't parse in HCL2
# (from https://github.com/amplify-education/python-hcl2/issues/13)

output "ebs_sizes" {
  value = {
    "x" : 600,
    "y" : 350,
    "z" : 350,
    "xx" : 200
  }
}