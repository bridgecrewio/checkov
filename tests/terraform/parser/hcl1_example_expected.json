{
  "output": {
    "ebs_sizes": {
      "value": {
        "x": 600,
        "y": 350,
        "z": 350,
        "xx": 200
      }
    }
  }
}